packages/ripple-nuxt-tide/modules/webform/components/TideContentRating.vue

```diff
@@ -250,7 +250,11 @@ $app-content-text-color: rpl-color('dark_neutral') !default;
   }
 
   .app-button--cancel {
-    background-color: rpl-color('danger');
+    background-color: $rpl-button-danger-background-color;
+  }
+
+  .app-button--cancel:hover {
+    background-color: $rpl-button-quick-exit-hover-background-color;
```

Comment:
Hey @Yuuupie can we add a new var called `$rpl-button-danger-hover-background-color` in "packages/components/Atoms/Global/scss/settings/_button.scss" ? Even it's same value, but easier for maintaining. Thanks!

Reply:
Yeah good idea